llvm/include/llvm-c/Core.h

```diff
@@ -744,6 +744,24 @@ LLVMModuleRef LLVMCloneModule(LLVMModuleRef M);
  */
 void LLVMDisposeModule(LLVMModuleRef M);
 
+/**
+ * Soon to be deprecated.
+ * See See https://llvm.org/docs/RemoveDIsDebugInfo.html.
```

Comment:
```suggestion
 * See https://llvm.org/docs/RemoveDIsDebugInfo.html#c-api-changes
```
Possible change here and below?